### PR TITLE
feat: decimal point options

### PR DESCRIPTION
**Pull request by ybouhjira**
[PRI-150]

### Description

add `maximumFractionDigits` and `minimumFractionDigits` options to `formatAmount` functions

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[PRI-150]: https://toptal-core.atlassian.net/browse/PRI-150?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ybouhjira** [comment]
@toptal-anvil ping reviewers

File: packages/picasso/src/utils/Formatters/format-amount.ts [hunk 1]
```diff
@@ -8,17 +8,28 @@ export interface FormatAmount {
   currency?: string
   /** Locale identifiers are case-insensitive ASCII. However, it's conventional to use title case (first letter capitalized, successive letters lower case) for script code, upper case for region codes, and lower case for everything else. */
   locale?: string
+  /**More options**/
+  options?: {
+    /** The minimum number of fraction digits to display **/
+    minimumFractionDigits?: number
+
+    /** The minimum number of fraction digits to display **/
```
Comment (LashaJini) [inline packages/picasso/src/utils/Formatters/format-amount.ts:16]:
```suggestion
    /** The maximum number of fraction digits to display **/
```

**Review by TomasSlama — COMMENTED** [review]
Hey 👋, thank you for taking time to contribute to Picasso, everything looks good to me. Only one thing, could you provide example in the [storybook](https://picasso.toptal.net/PRI-150-formatAmount/?path=/story/utils-formatters--formatters)

**Comment by ybouhjira** [comment]
Thanks @TomasSlama I will add it

File: .changeset/gentle-meals-sort.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+'@toptal/picasso': minor
+---
+
+---
+
+### formatAmount()
```
[the remark this answers is not in the document]
Comment (OndrejTuma) [reply .changeset/gentle-meals-sort.md:7]: maybe only `### Formatters` is enough and describe the method `formatAmount` in the list item?

File: packages/picasso/src/utils/Formatters/format-amount.ts [hunk 3]
```diff
@@ -8,17 +8,29 @@ export interface FormatAmount {
...
+  }
 }
 
+const DEFAULT_OPTIONS = {}
+
 export const formatAmount = ({
   amount,
   currency = DEFAULT_CURRENCY,
   locale = DEFAULT_LOCALE,
+  options = DEFAULT_OPTIONS,
```
Comment (OndrejTuma) [inline packages/picasso/src/utils/Formatters/format-amount.ts:26]: maybe the `DEFAULT_OPTIONS` here is not necessary, wdyt?

Comment (ybouhjira) [reply packages/picasso/src/utils/Formatters/format-amount.ts:26]: Yes you're right

**Comment by ybouhjira** [comment]
@TomasSlama 
image

**Review by TomasSlama — APPROVED** [review]
👍

**Comment by TomasSlama** [comment]
@toptal-anvil ping reviewers

**Review by LashaJini — APPROVED** [review]
Looks good. Thank you for contribution :bow:

**Review by OndrejTuma — APPROVED** [review]
Looks good to me 🙇 good job 💪

**Pull request merged**